search-service/src/test/kotlin/com/egm/stellio/search/service/AttributeInstanceServiceTests.kt

```diff
@@ -598,6 +592,41 @@ class AttributeInstanceServiceTests : WithTimescaleContainer {
         assertEquals("Attribute outgoing has an instance without an observed date", exception.message)
     }
 
+    @Test
+    fun `it should delete attribute instance`() {
+        val attributeInstance = gimmeAttributeInstance()
+        attributeInstanceService.create(attributeInstance).block()
+
+        runBlocking {
+            attributeInstanceService.deleteEntityAttributeInstance(
+                temporalEntityAttribute.id,
+                attributeInstance.instanceId
+            ).fold({ Assertions.assertTrue(it == null) }, { assertEquals(Unit, it) })
+        }
+
+        val enrichedEntity =
+            attributeInstanceService.search(TemporalQuery(), temporalEntityAttribute, false)
+
+        StepVerifier.create(enrichedEntity)
+            .expectNextMatches { it.size == 0 }
+            .expectComplete()
+            .verify()
+    }
+
+    @Test
+    fun `it should no delete attribute instance if instance are not found`() {
```

Comment:
are -> is (be careful, you often use wrongly the plural form). and also no -> not.